tests_new/integration_tests/modules/share_base/test_new_crossacc_s3_share.py

```diff
@@ -33,9 +27,9 @@
 )
 
 ALL_S3_SHARABLE_TYPES_NAMES = [
-    ShareableType.Table.name,
-    ShareableType.StorageLocation.name,
-    ShareableType.S3Bucket.name,
+    'DatasetTable',
+    'DatasetStorageLocation',
+    'S3Bucket',
```

Comment:
Thats' not right. We need enum.names and not enum.values
Should be 'Table', 'StorageLocation', 'S3Bucket'